.storybook/webpack.config.js

```diff
@@ -14,14 +15,24 @@ const { CheckerPlugin } = require("awesome-typescript-loader")
  */
 const {
   WEBPACK_DEVTOOL = "cheap-module-eval-source-map",
-  METAPHYSICS_ENDPOINT
+  METAPHYSICS_ENDPOINT,
+  USER_ID,
+  USER_ACCESS_TOKEN,
 } = env.config().parsed
 
 const sharifyPath = sharify({
   METAPHYSICS_ENDPOINT,
 })
 
-let plugins = [new CheckerPlugin()]
+const plugins = [new CheckerPlugin()]
+if (USER_ID && USER_ACCESS_TOKEN) {
+  plugins.push(new webpack.DefinePlugin ({
+    "process.env.USER_ID": JSON.stringify(USER_ID),
+    "process.env.USER_ACCESS_TOKEN": JSON.stringify(USER_ACCESS_TOKEN),
+  }))
+} else {
+  console.warn("\x1b[31m[!] Specify USER_ID and USER_ACCESS_TOKEN environment variables to use authenticated features.\x1b[0m")
```

Comment:
lol what code is this?

Reply:
This is an ANSI escape sequence which makes the text show red 😄